.travis.yml

```diff
@@ -0,0 +1,20 @@
+dist: xenial
+sudo: false
+language: python
+cache: pip
+
+install: pip install tox
+script: tox
+
+matrix:
+  include:
+    - python: "3.6"
+      env: TOXENV=py36
+    - python: "3.7"
+      env: TOXENV=py37
+    - python: "3.6"
+      env: TOXENV=black
```

Comment:
Should linters run first to give feedback earlier when a PR does not match expected quality standards?